index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- Navigation Bar-->
+    <header class="header-container">
+        <div class="left">
+            <a href="/blank"><img id="icons" src="./graphics/open-menu.png" alt="menu"></a>
+            <a href="https://www.youtube.com/"><img class="logo" src="graphics/yt_logo_rgb_light.png" alt="Main-logo"></a>
+
+        </div>
+        <div class="center">
+            <input type="text">
+            <button><img id="icons" src="./graphics/icons8-search-50.png" alt="Search"></button>
+        </div>
+        <div class="right">
+            <a href="/blank"><img class="navbar-icons" src="./graphics/upload.png" alt="upload"></a>
+            <a href="/blank"><img class="navbar-icons" src="./graphics/notification.png" alt="notification"></a>
+            <a href="/blank"><img class="user-account" src="./graphics/user.png" alt="user-avatar"></a>
+        </div>
+       
+    </header>
```

Comment:
- [ ] **[OPTIONAL]:** For semantic html it's a good ractice to use the <nav> tag for the navigation part